app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class ContactsController < Api::V1::ApiController
+      skip_before_action :authenticate_user!
+
+      def create
+        contact = Contact.create!(contacts_params)
+        AdminMailer.new_contact_question(contact)
```

Comment:
Maybe we can move this to the Contact model and call it on an after_create callback.

Reply:
It depends on the requirement, if we need a new_contact_question email to be sent every time we create a contact then the callback makes sense. But if we need the email to be sent only from this endpoint then the callback is not necessary.
Feel free to decide what's best, I don't how big is going to be the scope of this project/target.